windows/RNCNetInfoCPP/pch.h

```diff
@@ -1,14 +1,14 @@
-#pragma once
-
-#define NOMINMAX
-
-#include <hstring.h>
-#include <restrictederrorinfo.h>
-#include <unknwn.h>
-#include <windows.h>
-
-#include <winrt/Microsoft.ReactNative.h>
-#include <winrt/Windows.Foundation.h>
-#include <winrt/Windows.Foundation.Collections.h>
-#include <winrt/Windows.Foundation.Metadata.h>
-#include <winrt/Windows.Networking.Connectivity.h>
+﻿#pragma once
```

Comment:
revert changes here? Looks like all whitespace difference?

Reply:
I have reverted the changes here